en/02_Developer_Guides/03_Forms/03_Form_Templates.md

```diff
@@ -19,22 +19,22 @@ $field = new TextField(..);
 $field->setTemplate('MyCustomTextField');
 ```
 
-To override the template for CMS forms, the custom templates should be located in **/app/templates**. Front-end form templates can be located in **/app/templates** or in the active theme's **/templates** directory.
+To override the template for CMS forms, the custom templates should be located in `app/templates/`. Front-end form templates can be located in `app/templates/` or in the active theme's `templates/` directory.
 
 [notice]
 It's recommended to copy the contents of the template you're going to replace and use that as a start. For instance, if
 you want to create a `MyCustomFormTemplate` copy the contents of `Form.ss` to a `MyCustomFormTemplate.ss` file and 
 modify as you need.
 
-*The default Form.ss can be found in `/vendor/silverstripe/framework/templates/SilverStripe/Forms/Includes/`*
+*The default Form.ss can be found at `/vendor/silverstripe/framework/templates/SilverStripe/Forms/Includes/Form.ss`*
 [/notice]
 
-By default, Form and Fields follow the Silverstripe CMS Template convention and are rendered into templates of the same 
-class name (i.e EmailField will attempt to render into `EmailField.ss` and if that isn't found, `TextField.ss` or 
+By default, `Form` and the various `FormField` subclasses follow the Silverstripe CMS Template convention and are rendered into templates of the same
+class name (i.e `EmailField` will attempt to render into `EmailField.ss` and if that isn't found, `TextField.ss` or
```

Comment:
```suggestion
class name (i.e. `EmailField` will attempt to render into `EmailField.ss` and if that isn't found, `TextField.ss` or
```

Reply:
Done. I've done a search-and-replace for all instances of "i.e " across the entire docs so we don't run into this again.